docs/Contributing-to-Tachyon.md

```diff
@@ -7,11 +7,12 @@ group: Development
 
 Thank you for your interest in Tachyon! We greatly appreciate any new features or fixes.
 
+### toc
 * [Submitting Code](#submitting-code)
```

Comment:
I think some of the other toc have 4 space indenting. Maybe this should be changed to match?